packages/program-boilerplate/__tests__/trigger.test.ts

```diff
@@ -308,8 +301,12 @@ describe("triggerProgram", () => {
       };
       const result = triggerProgram(testBody, spyingProgram);
       expect(result).toStrictEqual({
-        json: {},
-        code: 204,
+        code: 500,
+        json: {
+          error:
+            "An error occurred in a webtask (PROGRAM_TRIGGER_VARIABLES_SCHEMA_REQUEST)",
+          message: undefined,
+        },
```

Comment:
TODO: I'm not sure if this was previously intended to return a 204 or if we just forgot to put the 500 in there